diesel.toml

```diff
@@ -0,0 +1,9 @@
+# For documentation on how to configure this file,
```

Comment:
This file is not nessesary same for `migrations` folder he already exist inside `bin/nanocld`